### PR TITLE
Remove WA BZ-2124379 [PodSecurity admission]

### PR DESCRIPTION
**Pull request by OdedViner**
Signed-off-by: OdedViner <oviner@redhat.com>

### CODE REVIEW

File: ocs_ci/utility/utils.py [hunk 1]
```diff
@@ -2579,9 +2579,6 @@ def create_rhelpod(namespace, pod_name, timeout=300):
     """
     # importing here to avoid dependencies
     from ocs_ci.helpers import helpers
-    from ocs_ci.ocs.utils import label_pod_security_admission
-
-    label_pod_security_admission(namespace=namespace)
```
Comment (petr-balogh) [inline ocs_ci/utility/utils.py:2584]: I think that you will need to verify with AWS RHEL UPI deployment before merging. For RHEL deployment we are using default project NS and the pod needs to have privileged access which I believe is not handled by OLM in this case and will fail if we are missing this piece of code.  So please verify this before removing.

You can check: 
https://url.corp.redhat.com/f3a9460
Which was used to verify https://github.com/red-hat-storage/ocs-ci/pull/6564

**Comment by ebenahar** [comment]
@OdedViner is there any ocs-ci workaround issue that tracks this change?

**Comment by OdedViner** [comment]
> @OdedViner is there any ocs-ci workaround issue that tracks this change?

@vavuthu @petr-balogh Did you open a github isusue?
@ebenahar I will open a new issue, if it is not exist...

File: ocs_ci/utility/utils.py (hunk 1 above)
Comment (OdedViner) [reply ocs_ci/utility/utils.py:2584]: https://ocs4-jenkins-csb-odf-qe.apps.ocp-c1.prod.psi.redhat.com/job/qe-trigger-test-pr/3719/
@petr-balogh

**Comment by petr-balogh** [comment]
I think that the W/A issue was not opened for this as it was not clear from start if this will be just documented to do this step or if it will be handled from product. So you can open new one issue with Workaround label.

**Comment by petr-balogh** [comment]
@OdedViner so as I said, the rhel pod with priviledged access is I think created in default NS. So I see from verfification job that it has failed:

```
failed on setup with "ocs_ci.ocs.exceptions.CommandFailed: Error during execution of command: oc -n default create -f /tmp/POD_cxiv7794 -o yaml.
Error is Error from server (Forbidden): error when creating "/tmp/POD_cxiv7794": pods "rhel-ansible" is forbidden: violates PodSecurity "restricted:latest": host namespaces (hostNetwork=true, hostPID=true), privileged (container "rhel" must not set securityContext.privileged=true), allowPrivilegeEscalation != false (container "rhel" must set securityContext.allowPrivilegeEscalation=false), unrestricted capabilities (container "rhel" must set securityContext.capabilities.drop=["ALL"]), runAsNonRoot != true (pod or container "rhel" must set securityContext.runAsNonRoot=true), runAsUser=0 (container "rhel" must not set runAsUser=0), seccompProfile (pod or container "rhel" must set securityContext.seccompProfile.type to "RuntimeDefault" or "Localhost")"
```

Which was done in this code:
https://github.com/red-hat-storage/ocs-ci/pull/6564/files

We can maybe improve this to make sure that if namespace != "openshift-storage" that we will not do this.

**Review by petr-balogh — APPROVED** [review]

**Review by vavuthu — APPROVED** [review]

**Pull request merged**